src/sections/layout/header/Header.tsx

```diff
@@ -26,11 +47,16 @@ export function Header() {
         logoImgSrc: logo
       }}>
       {user ? (
-        <Navbar.Dropdown title={user.name} id="dropdown-user">
-          <Navbar.Dropdown.Item href="#" onClick={onLogoutClick}>
-            {t('logOut')}
-          </Navbar.Dropdown.Item>
-        </Navbar.Dropdown>
+        <>
+          <Navbar.Dropdown title="Add Data" id="dropdown-addData">
+            {list}
+          </Navbar.Dropdown>
```

Comment:
I'd like to suggest a small improvement regarding the way we render the options for the "Add Data" button.

Currently, the code dynamically generates the dropdown items by mapping over the routesToRender array. While this approach is efficient and aligns with the DRY Principle (Don't Repeat Yourself), in this case, since the list is small and static, explicitly listing the items can enhance readability and comprehension.

The best practice depends on the context:

For static lists that don't change, or when each item requires distinct customization, explicitly listing items tends to be clearer.
For dynamic lists, especially those generated from data, mapping over an array is typically more efficient and maintainable.
In our situation, we fall under scenario 1, as there is no data fetching required for the dropdown options, the list is small, and unlikely to change. I would suggest using explicit listing for clarity:

```
<Navbar.Dropdown title="Add Data" id="dropdown-addData">
  <Navbar.Dropdown.Item href={`/spa${Route.DATASETS}`} disabled={true}>
    {t('navigation.newDataverse')}
  </Navbar.Dropdown.Item>
  <Navbar.Dropdown.Item href={`/spa${Route.CREATE_DATASET}`} disabled={false}>
    {t('navigation.newDataset')}
  </Navbar.Dropdown.Item>
</Navbar.Dropdown>

```

Please let me know if you have any questions or if there's a specific reason we prefer mapping over an array in this case. Thanks!